include/oneapi/dpl/pstl/glue_algorithm_ranges_impl.h

```diff
@@ -225,11 +226,12 @@ template <typename _ExecutionPolicy, typename _Range, typename _Compare>
 oneapi::dpl::__internal::__enable_if_execution_policy<_ExecutionPolicy, oneapi::dpl::__internal::__difference_t<_Range>>
 is_sorted_until(_ExecutionPolicy&& __exec, _Range&& __rng, _Compare __comp)
 {
+    auto __view = views::all_read(::std::forward<_Range>(__rng));
```

Comment:
In general this review looks very good to me. But I am slightly concerned about this line and the similar line above. Basically, you move arbitrary view that is passed here. Do we really want that behavior?

Reply:
Actually, it will work correctly, because we don't access to moved (__rng)  variable after moving.